src/quo2/components/community/community_view.cljs

```diff
@@ -1,16 +1,20 @@
 (ns quo2.components.community.community-view
-  (:require [quo2.components.community.style :as style]
+  (:require [quo.gesture-handler :as gesture-handler]
+            [quo2.components.community.style :as style]
             [quo2.components.icon :as icons]
             [quo2.components.markdown.text :as text]
             [quo2.components.tags.permission-tag :as permission]
             [quo2.components.tags.tag :as tag]
             [quo2.foundations.colors :as colors]
-            [quo.gesture-handler :as gesture-handler] ;;TODO move to quo2
-            [react-native.core :as rn]))
+            [quo2.theme :as theme]
+            [react-native.core :as rn]
+            utils.money))
 
 (defn community-stats
-  [{:keys [icon members-count icon-color]}]
-  [rn/view (style/stats-count-container)
+  [{:keys [icon members-count icon-color accessibility-label]}]
+  [rn/view
+   {:accessibility-label accessibility-label
```

Comment:
to make a view accessible you generally have to add `:accessibile true` to it as well. consider adding it.

Reply:
I agree with the technical importance of using `accessible`, but I don't think the mobile app cares about accessibility, like at all.

> When a view is marked as accessible, it is a good practice to set an accessibilityLabel on the view, so that people who use VoiceOver know what element they have selected.
>
> -- https://reactnative.dev/docs/accessibility#accessibilitylabel

We use accessibility labels for machines (e2e tests), not for humans. So in the end, if we really care about accessibility I think we (designers + devs) should go back to the drawing board and define some guidelines and ways of *testing accessibility* (e.g. using screen readers). It's a large topic for sure.

But as far as I remember @J-Son89, accessibility is not a priority until the product is launched. And in terms of color accessibility, I think our colors are not visually good for certain types of visual impairments (lack of contrast being an important one).

So in the end, why bother with `accessible` at this very moment if in the end the app is not accessible? 🤷🏼‍♂️

Reply:
Actually for e2e tests it's needed. For jest tests it's not. But yeah you're right probably it's unnecessary to add unless it's required for e2e :)